download_models.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
We can refactor the modelDownload.sh to handle both cases to help reduce confusion by having 2 model download scripts.

Reply:
Address in future PR